README.md

```diff
@@ -461,6 +461,11 @@ class mongodb::server {
 }
 ```
 
+##### `repl_enable_majority_read_concern`
+Use this setting to configure replication.enableMajorityReadConcern setting.
+For more information please refer to [MongoDB Read Concern "majority"](https://dochub.mongodb.org/core/psa-disable-rc-majority-3.6).
+Default: true
```

Comment:
This doesn't appear to match the default `undef`.

Reply:
Yeah, I struggled with that. According to the MongoDB docs, if you don't specify it (i.e. undef) then it defaults to true. So should my module documentation reflect that fact, or how I'm doing it in the module? This setting is for a very specific use case, so unless you it applies, most users will want to accept the default (i.e. undef which means it will not be in the config file which means it will be true, per the mongo docs).

Reply:
OK, so perhaps a better solution would be this: state that the default is undef and add parenthetically that this implies the MongoDB default of true. I will make it so.